projects/eval_scannet/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Files not ready for commit
+test_bboxes_3d.py
```

Comment:
do these work if they're in a subfolder like this?

Reply:
Yeah :) it's pretty useful for keeping them clean + porting across projects